utils/sccli.sh

```diff
@@ -80,19 +80,21 @@ function openshift()
     sudo sed -i.back "/^IMAGE=*/cIMAGE=\"${dockerRegistry}/${imageName}\:${imageTag}\"" /etc/sysconfig/openshift_option
 
     # ensure the docker containers are present
-    docker pull ${dockerRegistry}/${imageName}:${imageTag}
-    docker pull ${dockerRegistry}/${imageName}-haproxy-router:${imageTag}
-    docker pull ${dockerRegistry}/${imageName}-deployer:${imageTag}
-    docker pull ${dockerRegistry}/${imageName}-docker-registry:${imageTag}
-    docker pull ${dockerRegistry}/${imageName}-sti-builder:${imageTag}
+    if [[ "$(docker images -q  ${dockerRegistry}/${imageName}:${imageTag} 2> /dev/null)" == "" ]]; then
```

Comment:
We are just checking for one image here. Ideally we should be checking per image.